src/js/angular/core/directives/operations-statuses/operations-statuses.directive.js

```diff
@@ -0,0 +1,124 @@
+import {OPERATION_GROUP_TYPE} from "../../../models/monitoring/operations/operation-group";
+import {OPERATION_STATUS} from "../../../models/monitoring/operations/operation-status";
+import {OPERATION_TYPE} from "../../../models/monitoring/operations/operation-type";
+
+angular.module('graphdb.framework.core.directives.operationsstatuses', [])
+    .directive('operationsStatuses', operationsStatusesDirectives);
+
+operationsStatusesDirectives.$inject = ['$location'];
+
+function operationsStatusesDirectives($location) {
+
+    return {
+        restrict: 'E',
+        templateUrl: 'js/angular/core/directives/operations-statuses/templates/operations-statuses.html',
+        link: linkFunc,
+        scope: {
+            activeOperations: '='
+        }
+    };
+
+    function linkFunc(scope) {
+        scope.OPERATION_STATUS = OPERATION_STATUS;
+        scope.OPERATION_GROUP_TYPE = OPERATION_GROUP_TYPE;
+        scope.OPERATION_TYPE = OPERATION_TYPE;
+        scope.operationsSummary = {};
+
+        // =========================
+        // Public functions
+        // =========================
+        /**
+         * Getters for monitoring view url depending on <code>operationGroupType</code>.
+
+         * @param {string} operationGroupType - the value must be one of {@see OPERATION_GROUP_TYPE} options.
+         *
+         * @return {string} the url of the monitoring view.
+         */
+        scope.getOperationUrl = (operationGroupType) => {
+            if (OPERATION_GROUP_TYPE.QUERIES_OPERATION === operationGroupType) {
+                return 'monitor/queries';
```

Comment:
Maybe extract them to a `RedirectUrl` constant on top.

Reply:
Done